### PR TITLE
Fix file name with space

### PR DESCRIPTION
**Pull request by hsnfirooz**
In this PR, these issues are addressed:

- #31. This is done by adding a dynamic form for OOD, and better argument parser for the python script.
- #30. This is mainly for OOD users to have all the log in the OOD app.
- #29. If the file is not supported, the name of the file will be logged for better debugging.
- #26. Now deploying script supports `ood-dev` to easily install the ood sandbox for testing.

In addition, the `env.yml` has specific versions for `faster-whisper`, `ctranslate`, `whisperX`, `cuda`, and `pytorch` for easier environment building.

Last but not least, this PR adds a prefix to slurm `job_name` for easier statistics analysis using slurm database.

### CODE REVIEW

File: bin/deploy-data/ood/form.yml [hunk 1]
```diff
@@ -89,7 +45,4 @@ attributes:
     display: true
     help: |
       (Optional) Select the whisper model.
-    options:
-    - [ "default",  "large-v3" ]
-    - [ "v2",  "large-v2" ]
-    - [ "v3",  "large-v3" ]
\ No newline at end of file
+    options: #Will be filled with bin/deploy-data/update_supported_lang.py
```
Comment (ruokolt) [inline bin/deploy-data/ood/form.yml:48]: Should this be bin/deploy-data/update_ood_form_values.py?

Comment (ruokolt) [inline bin/deploy-data/ood/form.yml:48]: Should this be bin/deploy-data/update_ood_form_values.py?

Comment (hsnfirooz) [reply bin/deploy-data/ood/form.yml]: Thanks Teemu for noticing.

Comment (hsnfirooz) [reply bin/deploy-data/ood/form.yml]: Thanks for noticing. You are correct.

**Pull request merged**